openjdk/excludes/ProblemList_openjdk11-openj9.txt

```diff
@@ -22,6 +22,19 @@
 
 # jdk_beans
 
+
+java/beans/PropertyEditor/6380849/TestPropertyEditor.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestColorClass.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestColorClassJava.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestColorClassNull.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestColorClassValue.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestFontClass.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestFontClassJava.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestFontClassNull.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/PropertyEditor/TestFontClassValue.java https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
+java/beans/XMLEncoder/java_awt_ScrollPane.java https://github.com/adoptium/aqa-tests/issues/2848 macosx-all
```

Comment:
https://github.com/adoptium/aqa-tests/issues/2810 was moved to Adoptium infra repo https://github.com/adoptium/aqa-tests/issues/2810#issuecomment-901099534

If we have this issue in hyc, maybe we should open an infra issue internally.

Reply:
I will remove java/beans/XMLEncoder/java_awt_ScrollPane.java (related to [adoptium/aqa-tests#2848](https://github.com/adoptium/aqa-tests/issues/2848) for macosx-all) and double-check if the issue occurs in HYC. If it does, I will open an internal infrastructure issue.

Reply:
We may also want to test this on Openj9 Jenkins